src/main/java/org/chainoptim/desktop/features/client/controller/ClientOrdersController.java

```diff
@@ -82,12 +88,49 @@ private List<ClientOrder> handleOrdersResponse(Optional<List<ClientOrder>> order
             this.clientOrders = orders.get();
             fallbackManager.setLoading(false);
             System.out.println("Orders received: " + clientOrders);
+            setTableView();
             bindDataToTableView();
+            setEditEvents();
         });
 
         return clientOrders;
     }
 
+    private void setTableView() {
+        tableView.setEditable(true);
+        clientIdColumn.setCellFactory(TextFieldTableCell.forTableColumn(new IntegerStringConverter()));
+        productIdColumn.setCellFactory(TextFieldTableCell.forTableColumn(new IntegerStringConverter()));
+        quantityColumn.setCellFactory(TextFieldTableCell.forTableColumn(new FloatStringConverter()));
+        statusColumn.setCellFactory(TextFieldTableCell.forTableColumn());
+
+        tableView.setMaxHeight(Double.MAX_VALUE);
```

Comment:
You should identify the configuration that can be taken out of here and reused for the other tables we'll have, perhaps through a static method of a util class.